inventory/byo/hosts.example

```diff
@@ -78,5 +78,5 @@ ose3-etcd[1:3]-ansible.test.example.com
 
 # host group for nodes
 [nodes]
-ose3-master[1:3]-ansible.test.example.com openshift_scheduleable=False
+ose3-master[1:3]-ansible.test.example.com openshift_schedulable=False
```

Comment:
Ah, looks like it never made it in the docs :(

I do see it in the openshift-extras repo though (enterprise-3.0 branch)
```
jdetiber@localhost:~/git/github/openshift/openshift-extras/oo-install git:(enterprise-3.0)➤ git grep schedu
src/ooinstall/cli_installer.py:from the API.  By default this Node will be unscheduleable but this can be changed
src/ooinstall/install_transactions.py:        scheduleable = True
src/ooinstall/install_transactions.py:            scheduleable = False
src/ooinstall/install_transactions.py:        write_host(n, base_inventory, scheduleable)
src/ooinstall/install_transactions.py:def write_host(host, inventory, scheduleable=True):
src/ooinstall/install_transactions.py:        if not scheduleable:
src/ooinstall/install_transactions.py:            facts += ' openshift_scheduleable=False'
```